File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {

        int count = 0;
        int answer = 0;
        for(int num : nums) {
            if(num == 0) {
                answer = Math.max(answer, count);
                count = 0;
            } else {
                count++;
            }
        }
        
        return Math.max(answer, count);
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int count = 0;
        int answer = 0;
        for(int i=0; i<nums.size(); i++) {
            if(nums[i] == 0) {
                count = 0;
            } else {
                count++;
            }
            
            answer = max(answer, count);

        }

        return max(answer, count);
        
    }
};